oci/oci.go

```diff
@@ -515,7 +515,9 @@ func (r *Runtime) StopContainer(c *Container, timeout int64) error {
 	}
 	done := make(chan struct{})
 	go func() {
-		for {
+		ticker := time.NewTicker(time.Duration(timeout) * time.Second)
```

Comment:
isn't it https://github.com/kubernetes-incubator/cri-o/pull/602/files#diff-87558c1a90a59158e2c18a316e0c6693R532 ?